sources/packages/web/src/components/institutions/confirmation-of-enrollment/modals/ConfirmCOEModal.vue

```diff
@@ -63,14 +64,15 @@ export default {
       try {
         await ConfirmationOfEnrollmentService.shared.confirmCOE(
           props.locationId,
-          props.applicationId,
+          props.disbursementScheduleId,
         );
         toast.success("Confirmed", "Confirmation of Enrollment Confirmed!");
       } catch (error) {
-        toast.error(
-          "Unexpected error",
-          "An error happened while confirming the COE.",
-        );
+        let errorMessage = "An error happened while confirming the COE.";
+        if (error.includes(FIRST_COE_NOT_COMPLETE)) {
```

Comment:
If have a different way already in place for doing the same thing. I would avoid using a string in the middle of a string to define what is the error type. I saw now other areas of the code using in this way and would not recommend continue doing it.
The solution that I am mentioning is in place since save draft was added. Please take a look and we can discuss as needed.

![image](https://user-images.githubusercontent.com/61259237/153087496-8b905b96-06a1-4252-bdcc-65407b9b3e97.png)

Reply:
As discussed, using error type I am setting the custom message at the Vue side and throwing ApiProcessError from API side.